input/templates/my-account-edit-address.hbs

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{> common/htmlhead}}
+</head>
+
+<body>
+  <div class="darkbg hidden"></div>
+  {{> common/header}}
+  <div class="address-book">
+    <div class="container">
+      <div class="my-account-title">
+        <span class="my-account-title-text icon-user">{{i18n "my-account:myAccount.title"}}</span>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="address-book-wrapper">
+            <div class="row">
+              <div class="col-sm-3">
+                {{> myaccount/my-account-sidebar positionTwo=true}}
+              </div>
+              <div id="my-account-desktop-content" class="col-sm-9">
+                <div class="address-book-content edit-address-book-content">
```

Comment:
Some place around here it's missing the `<form>` object. Fill it the the fields `method="POST"` and `action="{{@root.meta._links.editMyAddressBook.href}}"` and some meaningful ID.
Put below a hidden input `addressId`, since we need that info from CTP to be able to change the address (http://dev.commercetools.com/http-api-projects-customers.html#change-address).
And another hidden input field for the CSRF Token, to avoid these kind of attacks.

Put all together, we need:
```
 <form id="form-edit-my-address-book" action="{{@root.meta._links.editMyAddressBook.href}}" method="POST">
    <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
    <input type="hidden" name="addressId" value="{{content.address.addressId}}"/>
```